base/src/main/kotlin/net/corda/v5/base/types/MemberX500Name.kt

```diff
@@ -16,32 +16,120 @@ import javax.security.auth.x500.X500Principal
  * The class also guaranties the reliable equality comparison regardless which order the attributes are specified when
  * parsing from the string or X500principal as well outputs the attributes to string in predictable order.
  *
- * @property commonName optional name by the which the entity is usually known. Used only for services (for
+ * Example usages:
```

Comment:
It would be useful to specify the constrains, what values are parsed, which are required, which are optional and in which order the name would be built. That exists in some form but it would be difficult to piece together the different bits to get the overall picture.